### ISSUE TITLE
ValueError when creating Encryptomatte node

### ISSUE THREAD

**Issue by fynnay** [Psyop/Cryptomatte#109]
### Summary

In *Nuke11.2v4*, creating new one node, while another node is selected, after previously having deleted an *Encryptomatte* a ValueError appears in the ScriptEditor.
I'm not sure exactly what other consequences this has, but it seems to prevent other *Encryptomatte* nodes from updating.

### Steps to reproduce

- Create a Constant node and select it
- Create an *Encryptomatte* node and delete it again
- Select the Constant node again
- Create another *Encryptomatte* node
- Error apprears in Script Editor (see below)

### Relevant logs and/or screenshots
```python
Traceback (most recent call last):
  File "/opt/applications/Nuke11.2v4/plugins/nuke/callbacks.py", line 127, in knobChanged
    _doCallbacks(knobChangeds)
  File "/opt/applications/Nuke11.2v4/plugins/nuke/callbacks.py", line 46, in _doCallbacks
    f[0](*f[1],**f[2])
  File "/data/users/fynn/.nuke/gizmos/thirdParty/Cryptomatte/cryptomatte_utilities.py", line 42, in <lambda>
    nuke.thisNode(), nuke.thisKnob()), nodeClass='Encryptomatte')
  File "/data/users/fynn/.nuke/gizmos/thirdParty/Cryptomatte/cryptomatte_utilities.py", line 446, in encryptomatte_knob_changed_event
    if unsafe_to_do_inputChange(node):
  File "/data/users/fynn/.nuke/gizmos/thirdParty/Cryptomatte/cryptomatte_utilities.py", line 535, in unsafe_to_do_inputChange
    return nuke.NUKE_VERSION_MAJOR > 7 and node.screenHeight() == 0
ValueError: A PythonObject is not attached to a node
```

**Comment by jonahfriedman** [comment]
I just reproduced this in 12.0v3, thank you for the report.

**Comment by jonahfriedman** [comment]
Fixed in the next version (1.2.5). It actually occurred on more than just these nodes, though it seems that it was harmless.
